File: Data/Maps/BookMap.cs
using Dapper.FluentMap.Dommel.Mapping;
using EntityModels;

namespace Data.Maps
{
    public class BookMap : DommelEntityMap<BookEM>
    {
        public BookMap()
        {
            ToTable("Book");
            Map(b => b.Id).IsIdentity();
            Map(b => b.Authors).Ignore();
        }
    }
}

File: DataVB/Maps/BookMap.vb
Imports Dapper.FluentMap.Dommel.Mapping
Imports EntityModels

Namespace Maps

    Public Class BookMap
        Inherits DommelEntityMap(Of BookEM)

        Public Sub New()
            ToTable("Book")
            Map(Function(b) b.Id).IsIdentity()
            Map(Function(b) b.Authors).Ignore()
        End Sub
    End Class

End Namespace

